_includes/manuals/1.8/4.4.7_prov_networking.md

```diff
@@ -0,0 +1,38 @@
+
+#### Host interface options
+
+* *Primary:* each host must have a single primary interface, which is used as the name of the host itself.  This should be considered the main hostname and would usually also carry the default route.
+* *Provision:* the interface on which provisioning of the operating system should be carried out.  This will be used for PXE (if applicable) with TFTP menus, running SSH finish scripts etc.  This may be different to the primary interface and should have access to Foreman and other provisioning systems.
+* *Managed:* whether Foreman orchestrates creation of DNS entries, DHCP reservations and configuration of the interface during provisioning.  Unmanaged interfaces would be used for informational purposes only.
+
+A simple, single-homed host would have one network interface with a DNS name set matching the hostname, then managed, primary and provision flags all ticked.  This would create one interface with DNS and DHCP records (if configured) over which the OS would be set up.
+
+A dual or multi-homed host could have one interface with primary enabled ("host.example.com") and another network with provision enabled ("host-build.example.com").  If both are also managed, Foreman will create DNS and DHCP records for both, but on the provision interface, the next-server/filename options for PXE will also be set.  A TFTP (PXELinux) menu would also be created for the provision interface's MAC address so the host can PXE boot on that physical interface, while its hostname would be assigned from the primary interface.
+
+#### Virtual machines and interfaces
+
+When Foreman deploys a host onto a compute resource, it creates a new interface on the VM for each interface specified when creating the host.
+
+After creation, Foreman reads back the network information and matches the created interfaces to the list of interfaces given for the host and stores the assigned MAC and IP addresses (depending on the compute resource type) in its database.  It then continues with orchestration, creating DNS and DHCP records etc. for the addresses retrieved from the new VM.  Once orchestration of these is complete, it powers up the VM.
+
+This design alleviates the need to supply MAC addresses for hosts being created on compute resources.
+
+#### Subnet options
+
+Subnets are defined in Foreman under *Infrastructure > Subnets*, and have a few options that affect how hosts are provisioned.
+
+* *IPAM:* DHCP will use a DHCP-enabled smart proxy, checking for assigned leases and reservations and suggesting a new IP from the range.  Internal DB will use Foreman's list of already-assigned IPs and doesn't rely on a DHCP smart proxy.  None disables auto-suggestion of IP addresses.
+* *Boot mode:* during OS provisioning, the template will configure the interface with either a static IP address or to use DHCP depending on the value of this setting.
+
```

Comment:
maybe add one more sentence

Various combinations of IPAM and Boot modes make sense but most common are DHCP (IPAM) with DHCP (Boot mode) and InternalDB with Static.